src/library_wasmfs.js

```diff
@@ -392,7 +395,60 @@ FS.createPreloadedFile = FS_createPreloadedFile;
         return __wasmfs_mknod(pathBuffer, mode, dev);
       }));
     },
-    // TODO: mkdev
+    major: (dev) => ((dev) >> 8),
+    minor: (dev) => ((dev) & 0xff),
```

Comment:
Are major and minor used?

Reply:
Not in any of the tests, but they were in the old API and were mentioned in the documentation, so I thought it might be important for compatibility. Other than that, there doesn't seem to be any usage that I could think of.

Reply:
Ah, in that case let's remove them. They might be dead code that was never removed. If we find a need later we'll add them then.